compose-tests/src/androidTest/java/radiography/test/compose/ComposeViewTest.kt

```diff
@@ -109,6 +117,53 @@ class ComposeViewTest {
     assertThat(boxView.height).isEqualTo(heightPx)
   }
 
+  @OptIn(UiToolingDataApi::class)
+  @Test
+  fun composeView_reports_call_chain_info() {
+    composeRule.setContentWithExplicitRoot {
+
+      @Composable
+      fun Call1() {
+        BasicText(text = "hello")
+      }
+
+      @Composable
+      fun Call2() {
+        Call1()
+      }
+
+      @Composable
+      fun Call3() {
+        Call2()
+      }
+
+      Column {
+        Call3()
+      }
+    }
+
+    val columnView = findRootComposeView()
+      .allDescendentsDepthFirst
+      .first { it.displayName == "Column" }
+
+    val columnChildren = columnView.children.toList()
+    assertThat(columnChildren).hasSize(1)
+
+    // Expect that the chain of custom Call functions is collapsed, so that the BasicText
+    // is represented with the name "Call3" but contains the semantic information of the text "hello".
+    val callGroup = columnChildren[0] as ComposeView
+    assertThat(callGroup.children.count()).isEqualTo(0)
+    assertThat(callGroup.displayName).isEqualTo("Call3")
+    assertThat(callGroup.semanticsConfigurations.firstNotNullOfOrNull { it[SemanticsProperties.Text] }?.map { it.toString() })
+      .containsExactly("hello")
+
+    assertThat(callGroup.callChain).hasSize(6)
+    assertThat(callGroup.callChain.map { it.name })
+      .containsExactly("Call3", "Call2", "Call1", "BasicText", "Layout", "ReusableComposeNode")
+    assertThat(callGroup.callChain.map { it.location?.sourceFile })
+      .containsExactly("ComposeViewTest.kt", "ComposeViewTest.kt", "ComposeViewTest.kt", "ComposeViewTest.kt", "BasicText.kt", "Layout.kt")
+  }
```

Comment:
This test seems to be testing several things (at least 2) and should be broken up to test one thing per test.

But also, this isn't testing any Radiography code, right? This test is making sure that the Compose behavior doesn't change? I don't know if this should exist here. Maybe yes, but then it should probably be in a separate test class since it actually tests compose.